django/contrib/auth/views.py

```diff
@@ -261,7 +261,9 @@ class PasswordResetConfirmView(PasswordContextMixin, FormView):
     @method_decorator(sensitive_post_parameters())
     @method_decorator(never_cache)
     def dispatch(self, *args, **kwargs):
-        assert 'uidb64' in kwargs and 'token' in kwargs
+        if 'uidb64' and 'token' not in kwargs:
```

Comment:
The new condition is not correct because it will not raise an exception for `kwargs` with `token` but without `uidb64`. Moreover it's rather a configuration issue so we should raise a `ImproperlyConfigured`, maybe:
```python
raise ImproperlyConfigured(
    'The URL path must contain 'uidb64' and 'token' parameters.'
) 
```
Can you add a test for this message?

Reply:
Thank you for the review, @felixxm. I will commit all proposed changes today.

Reply:
@mateoradman Thanks :+1: Let me kinow when you're ready for another round of reviews.

Reply:
@felixxm I am having doubts where to place the test for the message raised in this function. Can you please advise me where would it fit the most, and if there is no file where it would fit, can I create a new one and what is the naming convention for new test files.
Thanks :)

Reply:
Maybe in the `auth_tests` :thinking: , you can add a new path to the `urls.py`, e.g.
```
path('reset/missing_parameters/', views.PasswordResetConfirmView.as_view()),
```
and a test to the `auth_tests.test_views.PasswordResetTest`.

Reply:
Thank you very much. 
Ready for another round of reviewing! 😁